hellomama_registration/settings.py

```diff
@@ -14,7 +14,7 @@
 import djcelery
 import dj_database_url
 import mimetypes
-
+from hellomama_registration.utils import MSG_TYPES
```

Comment:
Addressed in https://github.com/praekelt/hellomama-registration/pull/84/commits/608a009e76e0d7d2b35bd64323095745a5f58270